diff --git a/node_modules/npm/node_modules/graceful-fs/polyfills.js b/node_modules/npm/node_modules/graceful-fs/polyfills.js
index 4c6aca7..efb662d 100644
--- a/node_modules/npm/node_modules/graceful-fs/polyfills.js
+++ b/node_modules/npm/node_modules/graceful-fs/polyfills.js
@@ -59,13 +59,13 @@ function patch (fs) {
   fs.fchmodSync = chmodFixSync(fs.fchmodSync)
   fs.lchmodSync = chmodFixSync(fs.lchmodSync)
 
-  fs.stat = statFix(fs.stat)
-  fs.fstat = statFix(fs.fstat)
-  fs.lstat = statFix(fs.lstat)
+  // fs.stat = statFix(fs.stat)
+  // fs.fstat = statFix(fs.fstat)
+  // fs.lstat = statFix(fs.lstat)
 
-  fs.statSync = statFixSync(fs.statSync)
-  fs.fstatSync = statFixSync(fs.fstatSync)
-  fs.lstatSync = statFixSync(fs.lstatSync)
+  // fs.statSync = statFixSync(fs.statSync)
+  // fs.fstatSync = statFixSync(fs.fstatSync)
+  // fs.lstatSync = statFixSync(fs.lstatSync)
 
   // if lchmod/lchown do not exist, then make them no-ops
   if (!fs.lchmod) {
